.github/workflows/nix-jupyterlab.yml

```diff
@@ -17,17 +17,14 @@ jobs:
         sudo apt autoclean -y || true
         docker rmi $(docker image ls -aq)
         df -h
-    - uses: cachix/install-nix-action@v7
-    - name: Setup repo
-      run: |
-        git submodule init && git submodule update
-    - uses: cachix/cachix-action@releases/v5
+    - uses: cachix/install-nix-action@v8
+    - uses: cachix/cachix-action@v6
       with:
         name: hasktorch
         signingKey: '${{ secrets.CACHIX_SIGNING_KEY }}'
         skipNixBuild: false
         file: experimental/jupyterlab/default.nix
```

Comment:
It doesn't need to be changed, but it might be better to do the same as hasktorch-skeleton.

Reply:
Hi Junji, I think this action needs to be redone soon anyway because the Jupyter Lab derivation will change. At the moment it looks like the workflow will be similar to running any other nix shell:
```sh
$ nix-shell ./nix/jupyter-shell.nix
```